# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/ipc/GfxMessageUtils.h
# Commit: ccfa767dba64
# Full Hash: ccfa767dba644dc193e0246eb8e8ff3377e8b8a5
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Client-side bindings mirror for precise CC, and merge similar codepaths. r=handyman
#   
#   * Context loss using RAII
#   * Move Program reflection Client-side
#   
# ==============================================================================

diff -r 71c122ac0ca7 -r ccfa767dba64 gfx/ipc/GfxMessageUtils.h
--- a/gfx/ipc/GfxMessageUtils.h	Wed Jan 08 22:19:14 2020 +0000
+++ b/gfx/ipc/GfxMessageUtils.h	Wed Jan 08 22:19:16 2020 +0000
@@ -37,7 +37,6 @@
 namespace mozilla {
 
 typedef gfxImageFormat PixelFormat;
-enum WebGLVersion : uint32_t { WEBGL1, WEBGL2 };
 
 }  // namespace mozilla
 
@@ -1180,13 +1179,6 @@
     : public BitFlagsEnumSerializer<mozilla::SideBits,
                                     mozilla::SideBits::eAll> {};
 
-template <>
-struct ParamTraits<mozilla::WebGLVersion>
-    : public ContiguousEnumSerializerInclusive<mozilla::WebGLVersion,
-                                               mozilla::WebGLVersion::WEBGL1,
-                                               mozilla::WebGLVersion::WEBGL2> {
-};
-
 } /* namespace IPC */
 
 namespace mozilla {